tests/integration/django/conftest.py

```diff
@@ -37,7 +37,11 @@ async def charm_file_fixture(
     pytestconfig: pytest.Config, ops_test: OpsTest, tmp_path_factory
 ) -> str:
     """Get the existing charm file."""
-    charm_file = await ops_test.build_charm(PROJECT_ROOT / "examples/django/charm")
+    charm_file = next(f for f in pytestconfig.getoption("--charm-file") if "django-k8s" in f)
```

Comment:
What's this django-k8s charm?

Reply:
This is related to `operator-workflows`. As there are many charms in the repository, integration tests are run with something like `tox -e integration ... --charm-file=./go/go-k8s.charm --charm-file=./django/django-k8s.charm --charm-file=./flask/flask-k8s.charm ...`. With this line it selects the correct charm file from the list of charms (if it exists). Otherwise the charm is built/packed twice.

Clearly I have a bug in here, as this raises StopIterator if the --charm-file option does not have the desired charm.